.github/workflows/pre-submit.e2e.generic.default.yml

```diff
@@ -1,7 +1,6 @@
 name: pre-submit e2e generic default
 
 on:
-  # TODO(github.com/slsa-framework/slsa-github-generator/issues/452): restore pre-submit
   # pull_request:
   #   branches: [main]
```

Comment:
Did you not actually want to re-enable the pre-submit?

Reply:
it's not working. I need to see whether that's because the `uses: ./.github/workflows/name` is always using main branch or not. I thought it was not, but I'm not sure anymore.
Will re-enable once I've validated everything works.